Makefile

```diff
@@ -4,6 +4,7 @@
 IS_DOCKER_INSTALLED = $(shell which docker >> /dev/null 2>&1; echo $$?)
 
 # docker info
+DOCKER_REGISTRY ?= docker.io
```

Comment:
Thanks for the PR @Frawless . 
We are pushing the multi-arch images using `docker buildx` in GitHub actions CI. It looks like buildx fail to push the images with the given default registry. (You can test this once by setting up buildx for push event)

Instead, we can use GitHub actions only to push the images. This will help to push faster and it will also provide the option to add the registry. You can check `docker/build-push-action@v2` action for the same.

Reply:
Have tested it on fork repo and it was able to push so I'm good with the changes. We can update it to use actions in upcoming enhancement.

Reply:
Ye it was working fine for me as well. So if I understand correctly you want to migrate from circle-ci to gh action for pushing the images? I wonder why it failed anyway because the default value is the same as it was without my changes?